### ISSUE TITLE
[Android] Latest release has overlay graphics appearing in double w/h?

### ISSUE THREAD

**Issue by khatharr** [libretro/RetroArch#10332]
![Screenshot_20200322-183503](https://user-images.githubusercontent.com/11417920/77271704-9462eb80-6c6c-11ea-9ccd-a6bed7ae4988.jpg)

The fast-forward, save/load notifications, and info messages are appearing with double the width and height they had in the previous release. This causes them to sometimes not fit onscreen.

**Comment by hizzlekizzle** [comment]
I believe that's related to the new 'auto scale graphics widgets' option located in settings > onscreen display > onscreen notifications. If you disable it, it should go back to the previous behavior, I think.

**Comment by khatharr** [comment]
Yes, that appears to function as a workaround. That's restored everything to the size it used to be.
